Complete drafts of all application slides

Albadylic opened the pull request:
- Added content for all slides
- Movie Data 2 is a bit of a diversion (it's using Git in the Terminal, which isn't an essential but is a helpful learning). We might prefer to skip a slide deck this week?
- They become less theory later on in the journey, most of the JS fundamentals come in the first four decks


I'm not sure how we'd like to facilitate the interview practice (final workshop). FAC tradition would say to self-organise in groups of three and swap between roles of interviewer, interviewee, and observer with time for feedback. Mentors could fill gaps, or give general tips and tricks? Would we need some example questions for the applicants? We don't usually share the questions themselves in advance but do share what they might be asked about.

S-adefe commented:
> * Added content for all slides
> * Movie Data 2 is a bit of a diversion (it's using Git in the Terminal, which isn't an essential but is a helpful learning). We might prefer to skip a slide deck this week?
> * They become less theory later on in the journey, most of the JS fundamentals come in the first four decks
> 
> I'm not sure how we'd like to facilitate the interview practice (final workshop). FAC tradition would say to self-organise in groups of three and swap between roles of interviewer, interviewee, and observer with time for feedback. Mentors could fill gaps, or give general tips and tricks? Would we need some example questions for the applicants? We don't usually share the questions themselves in advance but do share what they might be asked about.

As we just chatted about, I think the Git workshop is totally fine and helpful.

In terms of interview practice, we've just talked about our mentors showing an example interview to the class (incl. a code read) and then getting them to practice in threes - but maybe with interview criteria we're looking for up on the slides. Could you write this into the workshops?

Albadylic commented:
- Updated based on Sarah's Feedback
- I think the easiest way to provide criteria for the interviews will be to create a short handout, I can put this together by Week 8. The interview structure and guidance is [here](https://foundersandcoders.notion.site/Interviewing-at-Founders-and-Coders-31dfee868c3348b2b6bc80d55adb7724). I've removed the link to the guidance in the slides, in favour of providing a simpler version for support circle. This would still be emailed to anyone who applies and is offered an interview.
